modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/ExtensionMessageSource.java

```diff
@@ -345,4 +373,13 @@ protected void doInitialise() throws InitialisationException {
       throw new InitialisationException(e, this);
     }
   }
+
+  private byte getSourceTransactionalAction() {
+    try {
+      return toActionCode(sourceAdapter.getTransactionalAction());
+    } catch (MuleException e) {
+      LOGGER.error("An error occurred obtaining the Source Transactional Action, defaulting to [NONE]");
```

Comment:
why would this fail? why default to NONE (and basically disobey the user) instead of bubbling the exception?